.github/workflows/secrets.test-workspaces.yml

```
@@ -20,9 +20,10 @@ jobs:
       contents: read
       statuses: write
     steps:
-      - uses: actions/checkout@v5
+      - uses: actions/checkout@df4cb1c069e1874edd31b4311f1884172cec0e10
         with:
-          ref: ${{ github.event.workflow_run.head_sha }}
+          ref: ${{ github.event.workflow_run.head_sha }
```

actionlint (1.7.12)

[error] got unexpected EOF while lexing end marker }}, expecting '}'

(expression)